test/pthread/exit.c

```diff
@@ -0,0 +1,29 @@
+
+// @expect verified
+
+#include <pthread.h>
+
+int x = 1;
+
+void *t1(void *arg)
+{
```

Comment:
These curly braces are not according to our coding standard (see your main procedure below).